src/woo_publications/metadata/api/viewsets.py

```diff
@@ -3,24 +3,45 @@
 from drf_spectacular.utils import extend_schema, extend_schema_view
 from rest_framework import permissions, viewsets
 
-from ..models import InformationCategory
+from ..models import InformationCategory, Theme
 from .filters import InformationCategoryFilterSet
-from .serializer import InformationCategorySerializer
+from .serializer import InformationCategorySerializer, ThemeSerializer
 
 
 @extend_schema(tags=["informatiecategorieën"])
 @extend_schema_view(
     list=extend_schema(
-        summary=_("Retrieve every information category."),
-        description=_("Retrieve every information category."),
+        summary=_("all available information category."),
+        description=_(
+            "Returns a paginated result list of existing information categories."
+        ),
     ),
     retrieve=extend_schema(
-        summary="Retrieve a specific information category.",
+        summary="the specific information category.",
```

Comment:
```suggestion
        summary="Retrieve a specific information category.",
```
I think the original was fine

Let's go for capitalized summaries to keep it consistent.